.github/renovate.json

```
@@ -0,0 +1,38 @@
+{
+  "reviewers": ["m2en", "MikuroXina"],
+  "ignoreTests": false,
+  "automerge": true,
+  "dependencyDashboard": true,
+  "semanticCommits": "enabled",
+  "fetchReleaseNotes": true,
+  "enabledManagers": ["npm", "github-actions", "dockerfile", "docker-compose"],
+  "major": {
+    "automerge": false
+  },
+  "packageRules": [
+    {
+      "matchDatasources": ["npm"],
+      "labels": ["renovate: update(npm)"]
+    },
+    {
+      "matchDatasources": ["github-actions"],
+      "labels": ["renovate: update(gh-actions)"]
+    },
+    {
+      "matchDatasources": ["dockerfile", "docker-compose"],
+      "labels": ["renovate: update(docker)"]
+    }
+  ],
+  "npm": {
+    "stabilityDays": 3
+  },
+  "vulnerabilityAlerts": {
+    "enabled": true,
+    "automerge": true,
+    "labels": ["security"],
+    "assignees": [
+      "m2en", "MikuroXina"
+    ]
+  },
+  "schedule": ["every weekday"]
```

```suggestion
  "schedule": ["on the first day of the week"]
```
[ドキュメント](https://breejs.github.io/later/parsers.html#text) も見たところ, `weekday` は平日 (月曜日から金曜日) のことなのでこれまでの設定と同じにしたいというのと矛盾していると思います.

---

ちゃんと読めてなかった
Fix: [`69ab452` (#652)](https://github.com/approvers/OreOreBot2/pull/652/commits/69ab4525cba18faae8f94c1c6a2240d84ec64cd7)